### PR TITLE
Feature user can create deck

### PR DESCRIPTION
**Pull request by arjunkahlon**
Screen Shot 2022-09-12 at 3 57 02 PM
Screen Shot 2022-09-12 at 6 02 54 PM

### CODE REVIEW

**Comment by arjunkahlon** [comment]
Screen Shot 2022-09-12 at 6 03 50 PM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please include your modal form for allowing the user to name the new deck

image

**Comment by arjunkahlon** [comment]
add-deck-component
Screen Shot 2022-09-13 at 4 48 07 PM

File: server/index.js [hunk 1]
```diff
@@ -1,12 +1,47 @@
+/* eslint-disable no-unused-vars */
+
```
Comment (thebearingedge) [inline server/index.js]: Please remove this so we don't accidentally introduce unnecessary variables.

File: server/public/layout.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  margin: 0;
+}
+
+.decks-list-container {
+  padding: 1rem;
+}
+
+.hidden {
+  display: none;
+}
```
Comment (thebearingedge) [inline server/public/layout.css]: There are a few Bootstrap helper classes you can use to replace all of these.

File: client/app.jsx [hunk 3]
```diff
@@ -1,8 +1,32 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+// import AppContext from './lib/app-context';
```
Comment (thebearingedge) [inline client/app.jsx]: Please remove commented out lines

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great! Just a little cleanup

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**